connexion/utils.py

```diff
@@ -10,6 +10,7 @@
 import pkgutil
 import sys
 import typing as t
+from rstr import xeger
```

Comment:
This is an external dependency that will need to be added to https://github.com/spec-first/connexion/blob/main/pyproject.toml#L46.

It should possibly be optional (like `flask` and `swagger-ui`).